src/main/java/com/minecolonies/coremod/network/messages/TransferItemsRequestMessage.java

```diff
@@ -0,0 +1,154 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
+import com.minecolonies.coremod.util.BlockPosUtil;
+import com.minecolonies.coremod.util.InventoryUtils;
+import com.minecolonies.coremod.util.Log;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.tileentity.TileEntityChest;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.network.ByteBufUtils;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * Transfer some items from the player inventory to the Builder's chest
+ * Created: January 20, 2017
+ *
+ * @author xavierh
+ */
+public class TransferItemsRequestMessage  extends AbstractMessage<TransferItemsRequestMessage, IMessage>
+{
+    /**
+     * The id of the building.
+     */
+    private BlockPos buildingId;
+    /**
+     * The id of the colony.
+     */
+    private int      colonyId;
+    /**
+     * How many item need to be transfer from the player inventory to the building chest
+     */
+    private ItemStack      itemStack;
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)